lib/chef/cookbook/gem_installer.rb

```diff
@@ -66,8 +66,13 @@ def install
                 tf.close
                 Chef::Log.trace("generated Gemfile contents:")
                 Chef::Log.trace(IO.read(tf.path))
-                so = shell_out!("bundle install", cwd: dir, env: { "PATH" => path_with_prepended_ruby_bin })
-                Chef::Log.info(so.stdout)
+                # Skip installation only if Chef::Config[:skip_gem_metadata_installation] option is true
+                unless Chef::Config[:skip_gem_metadata_installation]
+                  # Add additional options to bundle install
+                  cmd = %Q{bundle install #{Chef::Config[:gem_installer_bundler_options]}}
```

Comment:
no, not %Q{}, but pass as an actual array.  that is supported on windows now as well, which would allow passing in both a single string "--local" as well as an arbitrary array of args `gem_installer_bundler_options = [ "--foo", "bar", "baz" ]`.

Reply:
that should break the shell_out expectation in the tests, and require changing it to `expect...  with("bundle", "install", "--local")`

oh probably need to `shell_out!(*cmd)` on the next line as well.

Reply:
Have changed cmd to [ "bundle", "install", Chef::Config[:gem_installer_bundler_options] ]